backend/de.metas.business/src/main/java/de/metas/location/interceptor/C_Country.java

```diff
@@ -45,6 +46,8 @@
 @Component
 public class C_Country
 {
+	final CountryService countryService = SpringContextHolder.instance.getBean(CountryService.class);
```

Comment:
your interceptor is a spring component,
so you don't have to use SpringContextHolder.instance.getBean.

Just get it from constructor param.
It will be wired automatically.